### PR TITLE
DEV: Change min_trust_to_flag_posts_voting_comments to group setting

### PR DESCRIPTION
**Pull request by martin-brennan**
New setting is flag_posts_voting_comments_allowed_groups

c.f. https://meta.discourse.org/t/changes-coming-to-settings-for-giving-access-to-features-from-trust-levels-to-groups/283408

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -19,3 +19,10 @@ plugins:
     default: 1
     enum: "TrustLevelSetting"
     client: true
+  flag_posts_voting_comments_allowed_groups:
```
Comment (Drenmi) [inline config/settings.yml:22]: Should we make `min_trust_to_flag_posts_voting_comments` hidden?

File: config/settings.yml [hunk 2]
```diff
@@ -19,3 +19,10 @@ plugins:
...
+    default: "11" # auto group trust_level_1
+    type: group_list
+    client: false
```
Comment (Drenmi) [inline config/settings.yml:25]: I think this is the default value. Do we want to be explicit about it?

**Review by Drenmi — APPROVED** [review]
💬

File: config/settings.yml (hunk 2 above)
Comment (martin-brennan) [reply config/settings.yml:25]: Ah good point, I will just remove :+1:

File: config/settings.yml (hunk 1 above)
Comment (martin-brennan) [reply config/settings.yml:22]: Yep will do :)

**Pull request merged**